main2.m

```diff
@@ -119,32 +130,35 @@
     %     %save meaninful data, estimated data, meaninful data no offset
     save(strcat('data/',experimentName,'/dataset.mat'),'dataset')
 end
+
 %% Data exploration
 % Plot ftDataNoOffset and/vs estimatedFtData
-for i=4:4
-    %     for i=1:size(input.ftNames,1)
-    FTplots(struct(input.ftNames{i},dataset.ftData.(input.ftNames{i}),strcat('estimated',input.ftNames{i}),dataset.estimatedFtData.(input.ftNames{i})),dataset.time);
-end
+if( scriptOptions.printPlots )
```

Comment:
Perfect, then I was just using the wrong starting point.  
I don't know if you have already wanted to do that, but I think it make sense to wrap in a function also the filtering (and perhaps the offset computations?) so we can use the same functions also in `computeEllipsoidWithGridDataset.m`.